File: 242-valid-anagram/valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if(len(s) != len(t)):
            return False
        else:
            s = sorted(s)
            t = sorted(t)
            for i in range(len(s)):
                if(s[i] != t[i]):
                    return False
            else:
                return True

File: 242-valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length()){
            return false;
        }else{
            int[] freqCount = new int[26];
            for(int i = 0;i<s.length();i++){
                freqCount[s.charAt(i) - 'a']++;
                freqCount[t.charAt(i) - 'a']--;
                }
            for(int i = 0;i<freqCount.length;i++){
                if(freqCount[i] != 0){
                    return false;
                }
            }
            return true;
            
            }
    }
}